companion/src/simulation/telemetrysimu.cpp

```diff
@@ -154,6 +156,55 @@ void TelemetrySimulator::onLogTimerEvent()
   logPlayback->stepForward(false);
 }
 
+void TelemetrySimulator::onGpsRunLoop()
+{
+    int a = ui -> gps_latlon -> text().contains(",");
+    if(not a){
```

Comment:
space after if and before bracket

Reply:
if (not a) {
Is that correct?

Reply:
Yes and just realised not should be '!' so 'if (!a) {'